util/cbfstool/flashmap/fmap.c

```diff
@@ -428,13 +428,13 @@ static int fmap_append_area_test(struct fmap **fmap)
 		goto fmap_append_area_test_exit;
```

Comment:
Can't point to the proper lines again: 417-420 should use `htoleXX` on the right side of `=`, except for `name`

Reply:
I'm not sure if this is really necessary. Those values are only used in lines 441-444, If I use htoleXX here, I would have to use leXXtoh in 441-444. This seems pointless.

Reply:
It has to be swapped twice. If someone were to attach a debugger to inspect the memory, he/she should be able to see exactly the same format as if it were written to a file between these two points. Also, these tests may change at some point, someone may add a detailed dump or actually write this data to a file. It is easier to fix the code now than to debug it later.